workers/tests/integrationtests/test_geodatabase.py

```diff
@@ -73,29 +73,20 @@ def test_get_feature_ids(self):
             asyncio.run(db_client.get_feature_ids("foo", self.fid_field))
 
     def test_get_area_of_bpolys(self):
-        result = asyncio.run(db_client.get_area_of_bpolys(self.bpolys))
+        result = asyncio.run(db_client.get_area_of_bpolys(self.feature.geometry))
         self.assertIsInstance(result, float)
 
-    def test_get_bpolys_from_db(self):
+    def test_get_region_from_db(self):
         result = asyncio.run(
-            db_client.get_bpolys_from_db(self.dataset, self.feature_id, self.fid_field)
+            db_client.get_region_from_db(self.feature_id, self.fid_field)
         )
         self.assertTrue(result.is_valid)  # GeoJSON object validation
 
         with self.assertRaises(UndefinedColumnError):
-            asyncio.run(
-                db_client.get_bpolys_from_db(self.dataset, self.feature_id, "foo")
-            )
+            asyncio.run(db_client.get_region_from_db(self.feature_id, "foo"))
 
         with self.assertRaises(DataError):
-            asyncio.run(
-                db_client.get_bpolys_from_db(self.dataset, "foo", self.fid_field)
-            )
-
-        with self.assertRaises(UndefinedTableError):
```

Comment:
Did you remove this error? Or why don't we test this *here* anymore?

Reply:
Now the function `get_bpolys_from_db()` operates only on the regions table. So this error can not be raised anymore. Before that when passing in an dataset which has not been present this error would have been raised.